packages/components/cascader-panel/src/node-content.tsx

```diff
@@ -19,17 +19,27 @@ export default defineComponent({
       required: true,
     },
     renderLabelFn: Function as PropType<RenderLabel>,
+    checkOnClickNode: Boolean,
+    showRadio: Boolean,
+    disabled: Boolean,
   },
-  setup(props) {
+  setup(props, { emit }) {
     const ns = useNamespace('cascader-node')
-    const { renderLabelFn, node } = props
+    const { renderLabelFn, node, showRadio, disabled, checkOnClickNode } = props
     const { data, label: nodeLabel } = node
 
     const label = () => {
       const renderLabel = renderLabelFn?.({ node, data })
       return isVNodeEmpty(renderLabel) ? nodeLabel : renderLabel ?? nodeLabel
     }
-
-    return () => <span class={ns.e('label')}>{label()}</span>
+    function handleClick() {
+      if (showRadio || disabled || !checkOnClickNode) return
```

Comment:
No need `showRadio` i think.

```suggestion
      if (disabled || !checkOnClickNode) return
```